docs/reference/style-guide.mdx

```diff
@@ -0,0 +1,412 @@
+```
+:::warning Tip title
+
+This is a tip.
+
+:::
+```
+
+:::note Markdown admonitions are sensitive to newlines
+
+Make sure you put a newline after the opening `:::` and before the closing `:::`.
+
+:::
+
+### Code blocks
+
+Surround code with three backticks to create a code block. Include the language after the first set of backticks to enable syntax highlighting.
+
+```bash
+echo "Hello, World!"
+```
+
+### Tables
+
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+```
+| Header 1 | Header 2 |
+| -------- | -------- |
+| Cell 1   | Cell 2   |
+
+### Blockquotes
+
+```
+> This is a blockquote.
+```
+> This is a blockquote.
+
+### Horizontal rules
+
+```
+---
+```
+---
+
+### `<details />`
+
+```
+<details>
+  <summary>Click to expand</summary>
+
+  This is the content of the details.
+</details>
+```
+
+<details>
+  <summary>Click to expand</summary>
+
+  This is the content of the details.
+</details>
+
+## Components
+
+### Admonition
+
+```mdx
+<Admonition type="tip">
+  This is a tip.
+</Admonition>
+```
+
+<Admonition type="tip">
+  This is a tip.
+</Admonition>
+
+### ActionCard and PrimaryCTA
+
+```
+<ActionCard
+  title="Get Started"
+  description="Learn how to get started with the Cloud Posse Reference Architecture"
+  action={{
+    label: 'Get Started',
+    href: '/reference/style-guide'
+  }}>
+  <PrimaryCTA to="/reference/style-guide">See our style guide</PrimaryCTA>
+</ActionCard>
+```
+
+<ActionCard
+  title="Get Started"
+  description="Learn how to get started with the Cloud Posse Reference Architecture"
+  action={{
+    label: 'Get Started',
+    href: '/reference/style-guide'
+  }}>
+  <PrimaryCTA to="/reference/style-guide">See our style guide</PrimaryCTA>
+</ActionCard>
+
+### AsciinemaWidget
+
+```
+<AsciinemaWidget src="https://asciinema.org/a/113643.json" />
+```
+
+<AsciinemaWidget src="https://asciinema.org/a/113643.json" />
+
+### AtmosWorkflow
+
+```
+<AtmosWorkflow workflow="deploy/organization" fileName="accounts" />
+<AtmosWorkflow workflow="deploy/organization" fileName="accounts" stack="foo"/>
+```
+
+<AtmosWorkflow workflow="deploy/organization" fileName="accounts" />
+
+<AtmosWorkflow workflow="deploy/organization" fileName="accounts" stack="foo"/>
+
+### CardGroup
+
+```
+<CardGroup>
+  <Card title="Card 1" description="Description 1" />
+  <Card title="Card 2" description="Description 2" />
+  <Card title="Card 3" description="Description 3" />
+</CardGroup>
+```
+
+<CardGroup>
+  <Card title="Card 1" description="Description 1" />
+  <Card title="Card 2" description="Description 2" />
+  <Card title="Card 3" description="Description 3" />
+</CardGroup>
+
+### CategoryList
+
+```
+<CategoryList path="/layers/spacelift/tutorials/"/>
+```
+
+<CategoryList path="/layers/spacelift/tutorials/"/>
+
+### CollapsibleText
+
+```
+<CollapsibleText title="This is a collapsible text">
+  This is the content of the collapsible text.
```

Comment:
The usage of collapsible text will be more obvious with more content. I suggest lorem ipsum.